src/content/docs/contributing/modules.md

```diff
@@ -49,17 +49,25 @@ If the module doesn't exist on `nf-core/modules`:
 
 We have implemented a number of commands in the `nf-core/tools` package to make it incredibly easy for you to create and contribute your own modules to nf-core/modules.
 
-1. Install the latest version of [`nf-core/tools`](https://github.com/nf-core/tools#installation) (`>=2.7`)
+1. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
+   <details>
+     <summary> 🛠️ Conda environment for the Module contribution workflow </summary>
+     If you use the conda package manager you can install all dependencies for the New module workflow directly into a new environment via
+     <code>
+       conda create -n nf-core -c bioconda "nextflow>=21.04.0" "nf-core>=2.7" nf-test prettier
```

Comment:
```suggestion
       conda create -n nf-core -c bioconda "nextflow>=21.04.0" "nf-core>=2.7" nf-test pre-commit
```

Reply:
`pre-commit` comes for free with nf-core. `prettier` has to be installed separately.